docs/crds/avinfrasetting.md

```diff
@@ -99,6 +101,13 @@ AviInrfaSetting integrates with Openshift Routes, via the annotation
     aviinfrasetting.ako.vmware.com/name: "my-infrasetting"
 ```
 
+### Using Namespace Annotation
+AviInfraSetting also integrates with all the obove objects - Services/Ingresses/Routes, via the Namespace annotation.
+
+```
+  annotations:
+    aviinfrasetting.ako.vmware.com/name: "my-infrasetting"
+```
```

Comment:
can we please elaborate more on this? Like will aviinfrasetting be applied to all ingress, svc objects present inside that namespace? What if ingress class and namespace both have aviinfrasetting? Which will take precedence?

Reply:
done, thanks.